infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -99,7 +99,7 @@
     // the files of the old instance
     private LogMetadata logMetadata;
 
-    private final long logSizeLimit;
+    private long logSizeLimit;
```

Comment:
Can we keep this as final ? maybe `initStreamLogDirectory` can return this number instead of assigning it inside the method